### PR TITLE
Hp procurve show mac address

### PR DESCRIPTION
**Pull request by dainok**
##### ISSUE TYPE
 - New Template Pull Request

##### COMPONENT
hp_procurve_show_mac-address.template

##### SUMMARY
Implementing "show mac-address" for HPE Procurve switches.

### CODE REVIEW

File: templates/hp_procurve_show_mac-address.template [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Value MAC ([0-9a-fA-F]{6}-[0-9a-fA-F]{6})
+Value PORT (\S+)
+Value VLAN (\d+)
+
+Start
+  ^.*MAC Address -> MAC
+
+MAC
+  ^\s+${MAC}\s+${PORT}\s+${VLAN} -> Record
```
Comment (FragmentedPacket) [inline templates/hp_procurve_show_mac-address.template]: Do you mind adding the following to the end of states for all templates:
```
  ^\s*$$
  ^. -> Error
```

It is possible implementing the above will cause errors as all lines need to be accounted for. If you have any questions regarding this, let me know and I'll be more than happy to work with you on explaining/fixing this.

Do you mind removing any templates not related to the command this PR is for? Also, please make sure all files/references to commands include all underscores.

These changes will need to be done on all PRs, but once tests pass and I take one last look, I'll get them merged in.

**Comment by dainok** [comment]
Removed not applicable files and added the `-> Error` as requested.

File: templates/hp_procurve_show_mac-address.template (hunk 1 above)
Comment (dainok) [reply templates/hp_procurve_show_mac-address.template]: Removed not applicable files and added the `-> Error` as requested.

Comment (dainok) [reply templates/hp_procurve_show_mac-address.template]: Everything should be fixed now.

File: templates/hp_procurve_show_mac-address.template [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Value MAC ([0-9a-fA-F]{6}-[0-9a-fA-F]{6})
+Value PORT (\S+)
+Value VLAN (\d+)
+
+Start
+  ^\s*--- -> Start_record
+
+Start_record
+  ^\s+${MAC}\s+${PORT}\s+${VLAN} -> Record
+  ^.* -> Error
```
Comment (FragmentedPacket) [inline templates/hp_procurve_show_mac-address.template]: Please update to:
```
  ^\s*$$
  ^. -> Error
```

Comment (dainok) [reply templates/hp_procurve_show_mac-address.template]: fixed as requested

**Pull request merged**